StripeConnect/StripeConnect/Source/EmbeddedComponentManager+Appearance.swift

```diff
@@ -80,9 +81,10 @@ extension EmbeddedComponentManager {
             ///   `CustomFontSource` when initializing the `EmbeddedComponentManager` before
             ///   referencing them in the appearance's `typography.font` property.
             public var font: UIFont?
-            /// The baseline font size set on the embedded component root.
-            /// This scales the value of other font size variables.
-            public var fontSizeBase: CGFloat?
+            /// The unscaled baseline font size set on the embedded component root.
+            /// This scales the value of other font size variables and is automatically scaled
+            /// when the component's size category is updated.
+            public var fontSizeBase: CGFloat? = 16
```

Comment:
I had to hardcode the default so that scaling was enabled by default. The only other option would be to send the default value from the web view on init.

This seems okay given that we put in our public docs that this has a default of 16pt